File: WebSolution/UIApp/src/app/tflportal/timesheet/view/models/TimeSheet.ts
import { TimeSheetDetails } from "./TimeSheetDetails";

export class Timesheet{
    constructor(public id:number,
                public status:string,
                public timesheetDate:string,
                public statusChangedDate:string,
                public employeeId:number){}
                
    public timeSheetDetails:TimeSheetDetails[]=[]
}

File: WebSolution/TFLOBL/Entities/TimesheetMgmt/TimeSheet.cs
namespace Transflower.TFLPortal.TFLOBL.Entities.TimesheetMgmt;


public class Timesheet
{
    public int Id { get; set; }
    public string? Status { get; set; }
    public int EmployeeId { get; set; }
    public DateTime TimesheetDate { get; set; }
    public DateTime? StatusChangedDate { get; set; }

    //navigational property
   public Employee? Employee { get; set; }
   public List<TimesheetEntry>? TimeSheetDetails { get; set; }

   
}
